ballsdex/packages/trade/cog.py

```diff
@@ -413,6 +427,64 @@ async def history(
         pages = Pages(source=source, interaction=interaction)
         await pages.start()
 
+    @coins.command(name="add")
+    async def coins_add(self, interaction: discord.Interaction, amount: int):
+        """
+        Add coins to your trade proposal
+        """
+        trade, trader = await self.check_trade_errors(interaction)
+        if not trade or not trader:
+            return
+
+        if trade:
+            if amount > await trader.fetch_player_coins():
+                await interaction.response.send_message(
+                    f"You don't have enough {settings.currency_name} to add that amount."
+                )
```

Comment:
```suggestion
            if amount < 0:
                await interaction.response.send_message(
                    "You cannot add a number below 0 as the amount.", ephemeral=True
                )
                return
            if amount > await trader.fetch_player_coins():
                await interaction.response.send_message(
                    f"You don't have enough {settings.currency_name} to add that amount.",
                    ephemeral=True,
                )
                return
```
adding a check so that the amount can't be negative, and those messages as ephemeral